src/modules/zenoh/zenoh.cpp

```diff
@@ -487,8 +489,10 @@ Zenoh demo bridge
 	PRINT_MODULE_USAGE_COMMAND("stop");
 	PRINT_MODULE_USAGE_COMMAND("status");
 	PRINT_MODULE_USAGE_COMMAND("config");
-	PX4_INFO_RAW("     addpublisher  <zenoh_topic> <uorb_topic>  Publish uORB topic to Zenoh\n");
-	PX4_INFO_RAW("     addsubscriber <zenoh_topic> <uorb_topic>  Publish Zenoh topic to uORB\n");
+	PX4_INFO_RAW("     add publisher  <zenoh_topic> <uorb_topic> <optional uorb_instance>  Publish uORB topic to Zenoh\n");
```

Comment:
I think "Publish uORB topic to Zenoh" and "Publish Zenoh topic to uORB" comments are the wrong way around.
Could verify this?

Reply:
It matches uxrce's convention, at least, and mirrors what was there to begin with. I think that this is intrinsically a somewhat cursed relationship to make, so I'm not sure that there's a right choice other than uxrce's convention?